meeting/meeting_june_8_2022.md

```diff
@@ -0,0 +1,56 @@
+# June 8th (6 PM CST)/9th (9 AM JST) Meeting 2022
+
+[A Telegram channel for people who ask you about where to join a channel](https://t.me/BitcoinDLCs)
+
+## Individual Updates (Sync)
+
+* **#Query(Nadav)**
+* **#Query(Chris)**
+* **#Query(Matt)**
+* **#Query(Tibo)**
+* **#Query(Lloyd)**
+* **#Query(Ben)**
+* **#Query(all)**
+
+## Implementation Updates (Sync)
+
+* bitcoin-s
+
+* rust-dlc
+
+* atomic finance
+
+* gun.fun
+
+* others?
```

Comment:
I believe the lava team will be on the call, perhaps they want to talk about their open source oracles.

https://github.com/lava-xyz/sybils